docs/guide/features.md

```diff
@@ -518,7 +518,7 @@ import MyWorker from './worker?worker'
 const worker = new MyWorker()
 ```
 
-The worker script can also use `import` statements instead of `importScripts()` - note during dev this relies on browser native support and currently only works in Chrome, but for the production build it is compiled away.
+The worker script can also use `import` statements (with `type: 'module'` option) instead of `importScripts()` - note during dev this relies on browser native support and currently only works in Chrome, but for the production build it is compiled away.
```

Comment:
This addition makes it sounds like you can only use `import` when using `type: 'module'` and that isn't the case. Could you expand on what you find confusing here?
Maybe it could be:
```suggestion
The worker script can also use `import` statements (both for classic and `type: 'module'` workers) instead of `importScripts()` - note during dev this relies on browser native support and currently only works in Chrome, but for the production build it is compiled away.
```

Reply:
Hmm but now it sounds like you could use import statement in classic type

Reply:
Yes, you can use it in classic types. It will be bundled and compiled away

Reply:
Oh I didn't know that. What is the point of the module type then?

Reply:
It works when `?worker` is used.
But it does not work in dev when `new Worker(new URL('./foo.js', import.meta.url))` is used (#8470).

Reply:
And @Pyrolistical, the module type controls how it is bundled in production (but should still be able to use ES modules to do your workers in both, except for the bug that @sapphi-red just linked)